source/JACDAC/JDProtocol.cpp

```diff
@@ -30,6 +30,8 @@ DEALINGS IN THE SOFTWARE.
 #include "Timer.h"
 #include "CodalDmesg.h"
 
+static char jacdac_name[CONTROL_PACKET_ERROR_NAME_LENGTH] = {'J', 'A', 'C', 'D', 'A', 'C'};
```

Comment:
should this be in protocol?

Reply:
Good question, made more sense the having it in the logic driver. I think it belongs in `JDProtocol`

Reply:
in every driver maybe?

Reply:
You'll be able to determine the driver based on the class in the control packet.

Reply:
What if I have multiple drivers with the same class on the same device?

Reply:
I'm resistant here because I think adding 6 bytes overall is not a big problem, but adding 6 bytes to every JACDAC driver is.

Reply:
JACDAC drivers already weigh in at 40 bytes; round 6 bytes to 10 for ManagedString overheads, and that's 50 bytes base RAM usage without any driver specific member variables.

Reply:
So be it. Btw, the device name is quite important to debug the network. Typically, you would think that the user would name the device and that name would be stored in flash.
Scenario: two accelerometers are on the network and the user names them "left" and "right" at configuration time. Then that name should be transmitted along with the control packets???

Reply:
So maybe that name should go beyond debug and actually be part of the logic driver control packet... and it has to be stored in flash :)

Reply:
Yeah, I'm still not sure if we want a serial number or a textual name. It does not make sense having both IMO.

Reply:
This can be a future PR, let's leave as is for now.